packit/config/package_config.py

```diff
@@ -254,6 +254,84 @@ def __eq__(self, other: object):
         )
 
 
+class PackageConfigValidation:
+    @staticmethod
+    def validate_primitive(package_config):
+        from packit.schema import PackageConfigSchema
+
+        return PackageConfigSchema().validate(
+            {
+                "config_file_path": package_config.config_file_path,
+                "specfile_path": package_config.specfile_path,
+                "downstream_package_name": package_config.downstream_package_name,
+                "upstream_project_url": package_config.upstream_project_url,
+                "upstream_package_name": package_config.upstream_package_name,
+                "upstream_ref": package_config.upstream_ref,
+                "upstream_tag_template": package_config.upstream_tag_template,
+                "dist_git_base_url": package_config.dist_git_base_url,
+                "dist_git_namespace": package_config.dist_git_namespace,
+                "spec_source_id": package_config.spec_source_id,
+                "create_pr": package_config.create_pr,
+                "actions": package_config.actions,
+            }
+        )
```

Comment:
Hasn't the `package_config` at this point been already validated ?
([get_packit_api()](https://github.com/packit-service/packit/blob/master/packit/cli/utils.py#L111) -> [get_local_package_config()](https://github.com/packit-service/packit/blob/8833635fdfbc681a518f75b7a1a363785a319ee2/packit/config/package_config.py#L296) -> [parse_loaded_config()](https://github.com/packit-service/packit/blob/8833635fdfbc681a518f75b7a1a363785a319ee2/packit/config/package_config.py#L360) -> [get_from_dict()](https://github.com/packit-service/packit/blob/8833635fdfbc681a518f75b7a1a363785a319ee2/packit/config/package_config.py#L176))

Anyway, something like `PackageConfigSchema().dump(package_config)` should create the dict for you.